.github/workflows/llama_cpp_plugin_build_and_test.yml

```diff
@@ -0,0 +1,80 @@
+name: llama_cpp_plugin_build_and_test
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - 'modules/llama_cpp_plugin/**'
+
+jobs:
+  build_ubuntu20:
+    runs-on: ubuntu-20.04
```

Comment:
It might be better to use a more powerful runner for the build job, something like `ubuntu-20.04-8-cores`, to reduce the build time.

Reply:
Done, I hope that the self-hosted runner pool is big enough to actually result in an improvement in the total runtime of the check over the github-hosted pool.